src/mono/netcore/System.Private.CoreLib/src/System/Threading/TimerQueue.Browser.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Threading
+{
+    // Runtime interface
+    internal static class WasmRuntime
```

Comment:
Why to separate class when it's all about Timer implementation. Should this be just part of `TimerQueue` ?

Reply:
This class was copy pasted from the current mono implementation. The runtime expects to find the icalls/callbacks in this class.